app/models/concerns/pay/chargeable.rb

```diff
@@ -34,5 +36,40 @@ def stripe_refund!(amount)
       update(amount_refunded: amount)
     end
 
+    if defined?(Receipts::Receipt) && required_receipt_attributes?
+      def receipt
+        Receipts::Receipt.new(
+          id: id,
+          product: Pay.config.application_name,
+          company: {
+            name: Pay.config.business_name,
+            address: Pay.config.business_address,
+            email: Pay.config.support_email,
+          },
+          line_items: line_items,
+        )
+      end
+
+      def line_items
+        line_items = [
+          ["Date",           created_at.to_s],
+          ["Account Billed", "#{owner.name} (#{owner.email})"],
+          ["Product",        Pay.config.application_name],
+          ["Amount",         ActionController::Base.helpers.number_to_currency(amount / 100.0)],
+          ["Charged to",     "#{card_type} (**** **** **** #{card_last4})"],
+        ]
+        line_items << ["Additional Info", owner.extra_billing_info] if owner.extra_billing_info?
```

Comment:
`extra_billing_info` will be something we will need to add to the User model. This is important for adding things like VAT IDs and such.

Reply:
We can add that field to this migration: https://github.com/jasoncharnes/pay/blob/master/db/migrate/20170503131610_add_fields_to_users.rb

Reply:
@excid3 how do you update the two schema.rb's (test/dummy/db and test/support) ?

Reply:
If I remember right, we just modify the schema.rb files? It's using an SQLite in-memory db for it, so I believe it just loads from the schema file.